src/components/base-address/BaseAddress.vue

```diff
@@ -244,15 +245,15 @@ export default defineComponent({
         localSchema.postalCode = origPostalCodeRules.concat([baseRules.zipCode])
         localSchema.region = origRegionRules
       } else {
-        localSchema.postalCode = origPostalCodeRules.concat([baseRules[AddressValidationRules.MAX_LENGTH](15)])
-        localSchema.region = [baseRules[AddressValidationRules.MAX_LENGTH](2), ...spaceRules]
+        localSchema.postalCode = origPostalCodeRules.concat([baseRules[AddressValidationRules.MAX_LENGTH]])
+        localSchema.region = [baseRules[AddressValidationRules.MAX_LENGTH], ...spaceRules]
```

Comment:
Old code looked weird. Does new code work as expected?

Reply:
The new code gives an error in my VS Code. The old code seems to be fine.

Reply:
Weird, I have the vice versa 👀

Reply:
The `(15)` and `(2)` are parameters to the function.

In your editor, if you hover on `baseRules`, does it appear as an object of functions?

Reply:
This was reverted back so I think it's OK. Resolving this.